cmd/protoc-gen-go-grpc/README.md

```diff
@@ -14,7 +14,8 @@ To restore this behavior, set the option `require_unimplemented_servers=false`.
 E.g.:
 
 ```sh
-  protoc --go-grpc_out=. --go-grpc_opt=require_unimplemented_servers=false[,other options...] \
+  protoc --go-grpc_out=. --go-grpc_opt=require_unimplemented_servers=false \
```

Comment:
@hanut19 this is not correct. Did you try running this? 

this is correct actually `protoc --go-grpc_out=. --go-grpc_opt=require_unimplemented_servers=false[,other options...] \`. It was fixed by this PR https://github.com/grpc/grpc-go/pull/6349